autofix-beta-apiversions.go

```diff
@@ -67,7 +123,11 @@ func main() {
 				os.Exit(1)
 			}
 			fmt.Printf("%s: autofixing %s/%s -> %s/%s\n", path, apiVersion, kind, newAPIVersion, kind)
-			lines[idx] = strings.Replace(line, apiVersion, newAPIVersion, 1)
+			newLine := fmt.Sprintf("apiVersion: %s", newAPIVersion)
+			if shouldCheckAPICapabilities[kind] {
+				newLine = wrapInCapabilitiesCheck(apiVersion, newAPIVersion)
```

Comment:
Is there anything that prevents an already-wrapped API version from being wrapped again on subsequent runs?

Reply:
There is now 😅